tests/Unit/Elliptic/DiscontinuousGalerkin/Test_ImposeInhomogeneousBoundaryConditionsOnSource.cpp

```diff
@@ -205,23 +204,17 @@ SPECTRE_TEST_CASE("Unit.Elliptic.DG.Actions.InhomogeneousBoundaryConditions",
     const ElementId<2> element_id{0, {{SegmentId{1, 0}, SegmentId{1, 1}}}};
     const domain::creators::Rectangle<Frame::Inertial> domain_creator{
         {{-0.5, 0.}}, {{1.5, 1.}}, {{false, false}}, {{1, 1}}, {{3, 3}}};
-    // Register the coordinate map for serialization
-    PUPable_reg(
-        SINGLE_ARG(domain::CoordinateMap<Frame::Logical, Frame::Inertial,
-                                         domain::CoordinateMaps::ProductOf2Maps<
-                                             domain::CoordinateMaps::Affine,
-                                             domain::CoordinateMaps::Affine>>));
 
     // Expected boundary contribution to source in element X:
-    //   ^ eta
-    // -18 0 12
-    //  -6 0  0
-    //  -6 0  0 > xi
+    //  ^ eta
+    // 18 24 36
+    //  3  0  0
+    //  0  0  0 > xi
     // This is the sum of contributions from the following faces (see 1D):
-    // - upper eta: [ -12 0 12 | -> xi
-    // - lower xi: | -6 -6 -6 ] -> eta
+    // - upper eta: [ 12 24 36 | -> xi
+    // - lower xi: | 0 3 6 ] -> eta
     // This expectation assumes the diagonal mass matrix approximation.
-    const DataVector source_expected{-6., 0., 0., -6., 0., 0., -18., 0., 12.};
+    const DataVector source_expected{0., 0., 0., 3., 0., 0., 18., 24., 36.};
```

Comment:
Why are you changing the numbers?

Reply:
Because @nilsdeppe requested I make the solution depend on dimensions